core/src/main/java/com/adobe/aem/commons/assetshare/components/actions/share/impl/EmailShareServiceImpl.java

```diff
@@ -230,14 +231,32 @@ private String getSignature(final SlingHttpServletRequest request) throws Reposi
      * @param userData the Map of data provided by the end-user (usually derived from the Request) to use in the email.
      * @return the protected Map; all String's are xss protected for HTML.
      */
-    private Map<String, Object> xssProtectUserData(Map<String, Object> userData) {
-        for(final Map.Entry<String, Object> entry : userData.entrySet()) {
-            if (entry.getValue() instanceof String) {
-                userData.put(entry.getKey(), xssAPi.encodeForHTML((String)entry.getValue()));
+    private Map<String, Object> xssProtectUserData(Map<String, Object> dirtyUserData) {
+    	    
```

Comment:
Run through code formatter

Reply:
ah yeah, started using a new version of eclipse forgot to add the formatter. I was going to use: https://github.com/Adobe-Consulting-Services/coding-standards